src/enrichment/lsst.rs

```diff
@@ -68,12 +69,99 @@ pub fn create_lsst_alert_pipeline() -> Vec<Document> {
+            rdkafka::config::ClientConfig::new()
+                // Uncomment the following to get logs from kafka (RUST_LOG doesn't work):
+                // .set("debug", "broker,topic,msg")
+                .set("bootstrap.servers", &kafka_producer_host)
+                .set("message.timeout.ms", "5000")
+                // it's best to increase batch.size if the cluster
+                // is running on another machine. Locally, lower means less
+                // latency, since we are not limited by network speed anyways
+                .set("batch.size", "16384")
+                .set("linger.ms", "5")
+                .set("acks", "1")
+                .set("max.in.flight.requests.per.connection", "5")
+                .set("retries", "3")
+                .create()
+                .unwrap();
+        Babamul { kafka_producer }
+    }
+
+    async fn process_alert(
+        &self,
+        alert: Document,
+        properties: Document,
+    ) -> Result<(), EnrichmentWorkerError> {
+        // Merge properties into the alert document
+        let mut merged = alert.clone();
+        merged.insert("properties", Bson::Document(properties.clone()));
+
+        // Determine if this alert is worth sending to Babamul
+        let min_reliability = 0.5;
+        let pixel_flags = vec![1, 2, 3];
+        let sso = false;
+
+        if (merged.get_f64("candidate.reliability").unwrap_or(0.0) < min_reliability)
+            || (merged
+                .get_document("candidate")
+                .unwrap()
+                .get_i32("pixel_flags")
+                .map(|pf| pixel_flags.contains(&pf))
+                .unwrap_or(false))
+            || (merged
+                .get_document("properties")
+                .unwrap()
+                .get_bool("rock")
+                .unwrap_or(false)
+                == sso)
+        {
+            // Skip this alert, it doesn't meet the criteria
+            return Ok(());
+        }
+
+        // Determine which topic this alert should go to
+        // Is it a star, galaxy, or none, and does it have a ZTF crossmatch?
+        // TODO: Get this implemented
+        // For now, all LSST alerts go to "babamul.none"
+        let category: String = "none".to_string();
+        let topic_name = format!("babamul.{}", category);
+        let schema = crate::filter::load_alert_schema().unwrap();
+        // Convert the merged document into an Alert
+        let alert_obj = Alert::from_bson_document(&merged).unwrap();
+        let _ = crate::filter::send_alert_to_kafka(
+            &alert_obj,
+            &schema,
+            &self.kafka_producer,
+            &topic_name,
+        )
+        .await
+        .unwrap();
+
+        Ok(())
+    }
+}
+
 pub struct LsstEnrichmentWorker {
     input_queue: String,
     output_queue: String,
     client: mongodb::Client,
     alert_collection: mongodb::Collection<mongodb::bson::Document>,
     alert_pipeline: Vec<Document>,
+    babamul: Option<Babamul>,
```

Comment:
the more I see conditional babamul stuff the more I feel like a rust feature would have made more sense. Let's keep it as is, and we can try to revisit that later if need be. What I like about the config based thing, is that the code doesn't become grayed out in the IDE like it would with a feature